README.md

```diff
@@ -1,24 +1,176 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h3><b>InqezWallet</b></h3>
+  <img src="./splash_mock.png" alt="InqezWallet screenshot" width="400" height="auto">
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About InqezWallet ](#-about-inqezwallet-)
+- [Live Demo](#live-demo)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👤 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [Features in progress working on ](#features-in-progress-working-on-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show Your Support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database creation
+# 📖 About InqezWallet <a name="about-inqezwallet"></a>
 
-* Database initialization
+InqezWallet is a budget tracking app designed to assist users in managing their finances efficiently. The app enables users to track their expenses, and budget goals effectively.
 
-* How to run the test suite
+# Live Demo
 
-* Services (job queues, cache servers, search engines, etc.)
+[Demo](https://inqez-wallet.onrender.com)
```

Comment:
- [ ] Please make sure you're live demo follows the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Currently, it looks like the screenshots below:
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/85ee826f-9ec1-4f50-8da7-18d2fc2252a6)
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/fe8eeda2-a938-4e12-a671-d12b7fe55e5a)
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/0c17a6ad-bda7-40f5-a10a-14333f58b5e9)